src/platform/common/types.ts

```diff
@@ -84,7 +84,6 @@ export interface IJupyterSettings {
     readonly debugCodeLenses: string;
     readonly debugpyDistPath: string;
     readonly stopOnFirstLineWhileDebugging: boolean;
-    readonly textOutputLimit: number;
```

Comment:
Had to trigger a code change for compiiler to kick in, fortunately this isn't used anywhere.